resources/lang/de/Template.properties

```
@@ -556,6 +556,9 @@ myAccountSave = "Speichern"
 myAccountSettings = "Kontoeinstellungen"
 myAccountShippingAddresses = "Lieferadressen"
 ; myAccount - My account
+; navigation
+menuLabel = "Menü"
```

Open mobile navigation menu
```suggestion
menuLabel = "Menü der mobilen Navigation öffnen"
```